stl/inc/bitset

```diff
@@ -68,25 +68,26 @@ public:
         size_t _Mypos; // position of element in bitset
     };
 
-    static _CONSTEXPR23 void _Validate(const size_t _Pos) noexcept { // verify that _Pos is within bounds
+private:
+    static constexpr void _Validate(const size_t _Pos) noexcept { // verify that _Pos is within bounds

```

Comment:
Simply making this `constexpr` can make `operator[](...)const` work in constexpr contexts. However, in cases where `_ITERATOR_DEBUG_LEVEL != 0` and index error happens, the error message will become terrible. The following piece of code will get something like `"cannot call non-constexpr function "_CrtDbgReport"..."` in debug mode:
```cpp
#include<bitset>
constexpr void _validate(int index) {
    _STL_VERIFY(index < 10, "");
}
constexpr bool _foo(int index) {
    _validate(index);
    return true;
}
int main() {
    constexpr auto b = _foo(20);
}
```
This can be solved by making `_Validate` do specialized checks in consteval context.

Reply:
Suggested implementation:
```cpp
    static void _Index_outside_range() noexcept {}

    static constexpr void _Validate(const size_t _Pos) noexcept { // verify that _Pos is within bounds
        if (_Is_constant_evaluated()) {
            if (_Pos >= _Bits) {
                _Index_outside_range();
            }
        } else {
#if _ITERATOR_DEBUG_LEVEL == 0
            (void) _Pos;
#else // ^^^ _ITERATOR_DEBUG_LEVEL == 0 / _ITERATOR_DEBUG_LEVEL != 0 vvv
            _STL_VERIFY(_Pos < _Bits, "bitset index outside range");
#endif // _ITERATOR_DEBUG_LEVEL == 0
        }
    }
```

Reply:
As commented previously, we would prefer to perform such changes in a followup PR. We really try to avoid "grab bag" PRs that change lots of unrelated things simultaneously, as they are difficult and risky to review.